forc-plugins/forc-node/src/local/op.rs

```diff
@@ -0,0 +1,53 @@
+use super::cmd::LocalCmd;
+use crate::{
+    chain_config::{check_and_update_chain_config, ChainConfig},
+    run_opts::{DbType, RunOpts},
+    util::HumanReadableCommand,
+};
+use anyhow::Context;
+use forc_tracing::println_green;
+use std::process::{Child, Command};
+
+/// Local is a local node suited for local development.
+/// By default, the node is in `debug` mode and the db used is `in-memory`.
+/// Returns `None` if this is a dry_run and no child process created for fuel-core.
+pub(crate) async fn run(cmd: LocalCmd, dry_run: bool) -> anyhow::Result<Option<Child>> {
+    check_and_update_chain_config(ChainConfig::Local).await?;
+
+    let run_opts = RunOpts::from(cmd);
+    let params = run_opts.generate_params();
+
+    let mut fuel_core_command = Command::new("fuel-core");
+    fuel_core_command.arg("run");
+    fuel_core_command.args(params.as_slice());
+
+    if dry_run {
+        println_green(&format!(
```

Comment:
It'd be nice to print the command even when it's not a dry run.

Reply:
Hmm that is an interesting idea! I think we should do it. Adding it.